pyglmnet/__init__.py

```diff
@@ -16,7 +16,7 @@
 # Dev branch marker is: 'X.Y.devN' where N is an integer.
 #
 
-__version__ = '1.2.dev0'
+__version__ = '1.1.1'
```

Comment:
If we do this, we'll have to make a full release. I'd rather just do it on the maint branch.